docs/_static/css/index.css

```diff
@@ -182,6 +182,11 @@ button {
   padding: 100px 3% 100px;
 }
 
+.hero-section {
+  position: relative;
+  z-index: -1;
+}
+
```

Comment:
I had to revert this change as it caused the buttons "Get started" and "API Reference" in the main page unclickable.